.vscode/extensions.json

```diff
@@ -0,0 +1,11 @@
+{
+	"recommendations": [
+		"EditorConfig.editorconfig",
+		"eamodio.gitlens",
+		"eg2.vscode-npm-script",
+		"wayou.vscode-todo-highlight",
+		"minhthai.vscode-todo-parser",
+		"ms-vscode.vscode-typescript-tslint-plugin",
+		"rbbit.typescript-hero"
```

Comment:
let's keep only te essentials, IMO would be editorconfig, eslint and prettier, right?

Reply:
Sure, this was created by default when we made a new RC app, so I just let it stay.